app/views/mission_control/jobs/jobs/_filters.html.erb

```diff
@@ -1,45 +1,55 @@
 <div class="filter level-left">
-  <div class="field is-grouped">
-    <div class="control">
-      <%= form_for :filter, url: application_jobs_path(MissionControl::Jobs::Current.application, jobs_status), method: :get,
-        data: { controller: "form", action: "input->form#debouncedSubmit" } do |form| %>
+  <%= form_for :filter, url: application_jobs_path(MissionControl::Jobs::Current.application, jobs_status), method: :get,
+    data: { controller: "form", action: "input->form#debouncedSubmit" } do |form| %>
 
+    <div class="field is-grouped">
+      <div class="control">
+        <%= form.label :job_class_name, class: "label" %>
         <div class="select is-rounded">
           <%= form.text_field :job_class_name, value: @job_filters[:job_class_name], class: "input", list: "job-classes", placeholder: "Filter by job class...", autocomplete: "off" %>
         </div>
+      </div>
 
+      <div class="control">
+        <%= form.label :queue_name, class: "label" %>
         <div class="select is-rounded">
           <%= form.text_field :queue_name, value: @job_filters[:queue_name], class: "input", list: "queue-names", placeholder: "Filter by queue name...", autocomplete: "off" %>
         </div>
+      </div>
 
-        <% if jobs_status == "finished" %>
+      <% if jobs_status == "finished" %>
+        <div class="control">
+          <%= form.label :finished_at_start, class: "label" %>
```

Comment:
I don't _love_ the default text of "Finished at start" that this leads to, but I figured not manually setting it so that it could presumably be overwritten via i18n might be nice. That said, I'm more than happy to change this and any of the others in whatever ways seem nicest.